test/server/enrollment-system.spec.js

```diff
@@ -0,0 +1,33 @@
+const chai = require('chai');
+chai.should();
+const enrollmentSystem = require('../../src/server/enrollment-system');
+
+describe('enrollment-system base tests', () => {
+  describe('validate the input is a non-empty Object', () => {
+    it('should return an empty object when nothing is passed in', () => {
+      const result = enrollmentSystem.veteranToSaveSubmitForm();
+      result.should.be.empty;
```

Comment:
don't think we can do this cause of the inability to use es2015 module import / export